server/db/testdata.js

```diff
@@ -133,28 +133,32 @@ const enterFakeData = (
       idConcept: 3,
     }))
     .then(() => User.create({
-      firstName: 'Amber',
-      lastName: 'Jones',
+      givenName: 'Amber',
+      familyName: 'Jones',
       totalExperiencePoints: 50,
       goal: 'to become a dog mother',
+      id: 1,
```

Comment:
Do we need this id? Sequalize auto increments, right?

Reply:
no, I had put that there because we was going to save the google id as the user id